noxfile.py

```diff
@@ -23,13 +23,17 @@
 import nox
 
 
-BLACK_VERSION = "black==19.10b0"
+# black 21.12b0 is the last version to support Python 2.7
+BLACK_VERSION = "black==21.12b0"
```

Comment:
Per https://github.com/psf/black/blob/main/CHANGES.md#2112b0.

Running with the latest version of black results in some Python 2 incompatible changes. This library has some unicode strings, and the distinction seems to matter for protobuf fields. (see https://developers.google.com/protocol-buffers/docs/proto3#scalar)